### DISCUSSION TITLE
Collection names not displaying correctly

### DISCUSSION THREAD

**Discussion by benjajorquera** [mmistakes/minimal-mistakes#2690] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.20.2
- Ruby gem or remote theme version: 2.6.6 (?)
- Jekyll version: 3.9
- Git repository URL: https://github.com/benjajorquera/benjajorquera.github.io
- Operating system: Windows

## Expected behavior

Show the name of the collection.

![imagen](https://user-images.githubusercontent.com/62187223/92961078-0c081b00-f445-11ea-9b68-8ae3d5de10eb.png)

## Steps to reproduce the behavior

I'm running this project locally, when configuring the site I wanted to add a list of posts separated by collections, so I added a collections-archive.md according to the documentation (in customization / layouts) similar to: https://github.com /mmistakes/minimal-mistakes/blob/master/docs/_pages/collection-archive.html

But the name of the collection (I only have the "portfolio" collection at the moment) is displayed as follows:

![imagen](https://user-images.githubusercontent.com/62187223/92960699-753b5e80-f444-11ea-971c-6fc6b056bcdb.png)

It seems that the code is transformed to highlighted. I do not know what to do.

**Comment by mmistakes** [comment]
Have a look at the theme's layout file for a collection. https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/collection-archive.html

```
{% capture label %}{{ collection.label }}{% endcapture %}
    {% if label != written_label %}
      <h2 id="{{ label | slugify }}" class="archive__subtitle">{{ label }}</h2>
      {% capture written_label %}{{ label }}{% endcapture %}
    {% endif %}
```

These lines are responsible for outputing the `<h2>` heading on the page. What's going on is it's pulling the collection's `label`, which is the key you use to set up the collection in `_config.yml`. Since those are typically set in lowercase, that's how it's being outputted.

What you can do is change it to `Portfolio` in the _config.yml file. But that's going to mess with some other things and you'll likely get it uppercased in the URL too.

You might want to look into adding a uppercase filter to `{{ label }}`, but that might not work for all titles. That's why I specifically didn't do it in the theme as it's rather opionated.

**Comment by benjajorquera** [comment]
Sorry I think I did not explain well. I have no problems with the name of the collection appearing in lowercase, what happens is that literally the name (the label, the header, html) appears "highlighted" on the page, for example when you want to show a code snippet.

![imagen](https://user-images.githubusercontent.com/62187223/92972240-4d55f600-f458-11ea-9a68-042a6e3dd5f0.png)

**Comment by mmistakes** [comment, marked as answer]
Ahh gotcha. Yeah I see your problem (and it's a common one).
You named the file `.md`. And Markdown treats indents as code blocks, which is why it highlighted the heading as a `<code>` block.

Either remove the indents, or renamed the file to `.html` like I'm doing with the demo files.

**Comment by benjajorquera** [comment]
Got it, thanks
